### PR TITLE
List

### PR DESCRIPTION
**Pull request by JatuFaique**
Please review list component

### CODE REVIEW

**Comment by JatuFaique** [comment]
lists

**Review by sonishreyas — APPROVED** [review]
Looks good to merge

File: components/lists.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+.list > li {
+  font-size: 0.95rem;
+  list-style: none;
+  margin: 1em;
+  margin-left: 0.5em;
+  cursor: pointer;
+}
+
+.list > li:hover {
+  color: #02705c;
```
Comment (pratik-kumar-sahu) [inline components/lists.css:54]: - You can use colors as variables.

**Review by pratik-kumar-sahu — APPROVED** [review]
- Looks good to merge.

**Pull request merged**